miqa/core/migrations/0013_auto_20210917_1235.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.7 on 2021-09-17 12:35
```

Comment:
If this file is auto-generated, should we be versioning it?

If it is not auto-generated, shouldn't it be given a meaningful name?